careless/stats/ccpred.py

```diff
@@ -47,6 +47,18 @@ def __init__(self):
             action="store_true",
             help="Pool all prediction mtz files into a single calculation rather than treating each file individually.",
         )
+        
+        self.add_argument(
```

Comment:
If arguments like this are to be added, I would request that they are added in `careless/states/parser.py:BaseParser` and then propagated to all the stats submodules.